src/quantum/README.rst

```diff
@@ -0,0 +1,161 @@
+Microsoft Azure CLI 'quantum' Extension
+==========================================
+
+Azure Quantum is the first open Quantum computing platform. It offers a range of services
+from quantum hardware to full-state simulators and quantum inspired optimizations,
+providing developers and customers access to the most competitive quantum offering
+on the market.
+
+To learn more about azure quantum visit:
+https://azure.microsoft.com/en-us/services/quantum/
+
+To learn more about quantum computing and Microsoft's Quantum Development Kit visit:
+https://docs.microsoft.com/quantum/
+
+
+# Using the `az quantum` extension to list and manage jobs in Azure Quantum
+
+1. Log in to Azure using your credentials.
```

Comment:
I suggest add one step, how to install quantum.
```
az extension add --name quantum
```

Reply:
To be added when we re-write this file as an RST. Thanks for the suggestion!